src/MessagePack/Formatters/FrozenCollectionFormatters.cs

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if NET8_0_OR_GREATER
+
+using System.Collections.Frozen;
+using System.Collections.Generic;
+using MessagePack.Formatters;
+
+#pragma warning disable SA1402 // File may only contain a single type
+#pragma warning disable SA1649 // File name should match first type name
+
+namespace MessagePack.ImmutableCollection
+{
+    public sealed class FrozenDictionaryFormatter<TKey, TValue> : IMessagePackFormatter<FrozenDictionary<TKey, TValue>?>
+        where TKey : notnull
+    {
+        private readonly IEqualityComparer<TKey>? comparer;
+
+        public FrozenDictionaryFormatter()
+        {
+            comparer = default;
```

Comment:
No need to initialize fields to their default value.
```suggestion
```

Reply:
Yes. It is unneccessary.